docs/releasenotes/1.3_migration_guide.md

```diff
@@ -0,0 +1,93 @@
+---
+layout: default
+title: v1.3.0 Migration Guide
+parent: pages.release_notes
+nav_order: 1
+---
+
+# v1.2.0 Migration Guide
+{: .no_toc }
+
+## Table of contents
+{: .no_toc .text-delta }
+
+1. TOC
+{:toc}
+
+---
+
+## Overview
+
+Hyperledger FireFly v1.3.0 is a feature release that includes changes around event streaming, contract listeners, define/publish APIs as well as a range of general fixes.
+
+For the most part, upgrading from v1.2.x to v1.3.0 should be a seamless experience, but there are several important things to note about changes between the two versions, which are described in detail on this page.
+
+## API definition/publication considerations
+
+**As of FireFly v1.3.0, by default, contract interfaces, contracts APIs, and token pools are unpublished when they are created.**
+
+These following described changes impact contract interfaces, contract APIs, and token pools. Previously, when creating one of the affected resources, if successful, the resource would be automatically peered to other members of the multi-party network. This default behaviour has changed, now when one of those resources is defined, it by default is not peered to other members of the network, unless a publish query parameter has been provided. 
+
+In FireFly v1.2.0 to create one of the affected resources and publish it to other parties, a `POST` call would be made to its respective API route and the broadcast would happen immediately. To achieve the same behaviour in FireFly v1.3.0, there are 2 options for all impacted resources, either providing a query parameter at creation to signal immediate publish, or a subsequent API call to publish the resources.
+
+> **NOTE**: All of the impacted resources can be deleted locally using `DELETE` APIs, however, once the resources has been published, it is no longer possible to delete.
```

Comment:
This is actually a pretty important new feature which I think should be highlighted a bit more prominently. Previously, if you made a mistake when creating a token pool (as an example) you could not create another one with the same name, etc so you were stuck with a broken REST resource. Now you have a chance to create it and use it on your FireFly node before publishing it and pinning it to the blockchain.

Reply:
On these points I've opted for a slight-rewrite in this commit https://github.com/hyperledger/firefly/pull/1461/commits/c137e9d33fcf11199d2dd2cd4cb7b58dec7c0656, will request a re-review when we're in a good place.